### ISSUE TITLE
[BUG] Config parameters are ignored by the connected Google Analytics 4 tag for the Universal Analytics property

### ISSUE THREAD

**Issue by raebo1017** [datacommonsorg/website#1715]
### Describe the bug
The connected Google Analytics 4 tag ignores any config parameters that customize data collection for the Universal Analytics property.  For the parameters we added to configs in DC prod account (which is a Universal Analytics property), these data won’t be sent to Google Analytics. [About connected site tags.](https://support.google.com/analytics/answer/9973999#considerations)
![image](https://user-images.githubusercontent.com/98435133/184062035-139a45a0-1a85-4b04-9525-a1236cee8c36.png)

### To Reproduce
1. Open the DC prod website and view a random page
2. Open the Google Analytics DC prod account and go to the real-time report. https://analytics.google.com/analytics/web/#/p249581446/realtime/overview?params=_u..pageSize%3D25%26_u..nav%3Dmaui&collectionId=life-cycle
3. Click page_view event in the real-time report
The parameters of page_event do not include the "client_storage", "anonymize_ip" or "place_cateogry".

### Expected behavior
"client_storage", "anonymize_ip"  and "place_cateogry" parameters should be included in all the events. It works in the autopush account but doesn't work in the prod account.

### Screenshots
Prod account page_view parameters:
![image](https://user-images.githubusercontent.com/98435133/184063034-9776d278-0d7a-4449-8a69-31a177b1ef6a.png)
Autopush account page_view parameters:
![image](https://user-images.githubusercontent.com/98435133/184063074-aaae28e9-4130-468f-b2d5-d06b0f52b92d.png)

### Solution
We can mitigate this issue by moving the parameters assigned in the gtag(‘config’) command for Universal Analytics to a gtag(‘set’) command before the gtag(‘config’) command in the file [base.html](https://github.com/datacommonsorg/website/blob/1e6e78400d3b137c1a5f3ba1fd37025052ba1a02/server/templates/base.html).
gtag(‘set’) allows you to set values that persist across all the subsequent gtag() calls on the page. [Google tag API reference](https://developers.google.com/tag-platform/gtagjs/reference).
Example:
`gtag('set', {<parameter-value-pair>, <parameter-value-pair>})`.

**Comment by shifucun** [comment]
Do we need to change prod GA to use "Google Analytics 4"? Also I remember that the Universal Analytics property will be deprecated in 2023?

**Comment by raebo1017** [comment]
I did more research on this and found we may already upgrade that to GA4.[How to check property type ](https://support.google.com/analytics/answer/11583832?hl=en). 
![image](https://user-images.githubusercontent.com/98435133/184195913-96fd1a0c-dd3d-4a62-a961-c86cb24ef48a.png)
I guess what we did is that we created a new GA4 property, and a connected tag is used to collect data in parallel with our existing Universal Analytics property. So an additional gtag(‘config’, ‘G-249581446’) call is made (without any parameters). Reference: [Add a Google Analytics 4 property (to a site that already has Analytics)](https://support.google.com/analytics/answer/9744165)
